2017-10-23 12:44:56,191-__main__-ERROR-setting\configs\cvonline.json:
    =>FOR_UNIT_TEST:False
    =>SPEC_ONLINE:False
    =>MODBUS_PORT:None
    =>LOG_LEVEL:ERROR
    =>RAISE_EXCEPTION:True
    =>DB_PARAMETER:{}
    =>FRAME_CORE:(1296, 972)
    =>LED_PORT:com7
    =>CONFIGS_DIR:setting\configs\
    =>THIN_FIBERS:('200/220', '105/125')
    =>IMG_ZOOM:2
    =>PDF_PARAMETER:{}
    =>SAVE_TEMP_IMG:False
    =>FRAME_SIZE:(1944, 2592)
    =>VIEW_LABEL:ManualCV
    =>OCTAGON_FIBERS:('10/130(oc)', '20/130(oc)')
    =>CAPILLARY:('capillary',)
    =>DYNAMIC_CAMERA:True
    =>SAVED_VIEW_ITEMS:{}
    =>BASE_DIR:D:\MyProjects\WorkProject\opencv4fiber\geomety
2017-10-23 12:44:58,223-__main__-ERROR- main info: ManualCV cvonline 
D:/MyProjects/WorkProject/opencv4fiber/geomety/main.py
2017-10-23 12:49:02,901-setting.parameter-ERROR-key update :
    =>cladRange:[500, 900]
    =>coreRange:[20, 140]
    =>fiberType:G652
    =>corepoint:[1296, 972]
    =>thresholdSize:{u'core': 60, u'clad': 175}
    =>ampPixSize:0.0792393
    =>imgsize:[1944, 2592]
    =>medianBlur:{u'corefilter': 15, u'cladfilter': False}
2017-10-23 12:49:02,901-pattern.classify-ERROR-meta_logger-amp:0.0792-thr:60,175-filter:15,False
2017-10-23 12:49:02,901-pattern.classify-ERROR-get fiber type: G652 - <pattern.classify.DoubleCircleClassify object at 0x08CDE490>
2017-10-23 12:49:12,711-__main__-ERROR-setting\configs\cvonline.json:
    =>FOR_UNIT_TEST:False
    =>SPEC_ONLINE:False
    =>MODBUS_PORT:None
    =>LOG_LEVEL:ERROR
    =>RAISE_EXCEPTION:True
    =>DB_PARAMETER:{}
    =>FRAME_CORE:(1296, 972)
    =>LED_PORT:com7
    =>CONFIGS_DIR:setting\configs\
    =>THIN_FIBERS:('200/220', '105/125')
    =>IMG_ZOOM:2
    =>PDF_PARAMETER:{}
    =>SAVE_TEMP_IMG:False
    =>FRAME_SIZE:(1944, 2592)
    =>VIEW_LABEL:ManualCV
    =>OCTAGON_FIBERS:('10/130(oc)', '20/130(oc)')
    =>CAPILLARY:('capillary',)
    =>DYNAMIC_CAMERA:True
    =>SAVED_VIEW_ITEMS:{}
    =>BASE_DIR:D:\MyProjects\WorkProject\opencv4fiber\geomety
2017-10-23 12:49:14,477-__main__-ERROR- main info: ManualCV cvonline 
D:/MyProjects/WorkProject/opencv4fiber/geomety/main.py
2017-10-23 12:49:15,868-setting.parameter-ERROR-key update :
    =>cladRange:[500, 900]
    =>coreRange:[20, 140]
    =>fiberType:G652
    =>corepoint:[1296, 972]
    =>thresholdSize:{u'core': 60, u'clad': 175}
    =>ampPixSize:0.0792393
    =>imgsize:[1944, 2592]
    =>medianBlur:{u'corefilter': 15, u'cladfilter': False}
2017-10-23 12:49:15,868-pattern.classify-ERROR-meta_logger-amp:0.0792-thr:60,175-filter:15,False
2017-10-23 12:49:15,868-pattern.classify-ERROR-get fiber type: G652 - <pattern.classify.DoubleCircleClassify object at 0x08EE5630>
2017-10-23 12:49:58,651-__main__-ERROR-setting\configs\cvonline.json:
    =>FOR_UNIT_TEST:False
    =>SPEC_ONLINE:False
    =>MODBUS_PORT:None
    =>LOG_LEVEL:ERROR
    =>RAISE_EXCEPTION:True
    =>DB_PARAMETER:{}
    =>FRAME_CORE:(1296, 972)
    =>LED_PORT:com14
    =>CONFIGS_DIR:setting\configs\
    =>THIN_FIBERS:('200/220', '105/125')
    =>IMG_ZOOM:2
    =>PDF_PARAMETER:{}
    =>SAVE_TEMP_IMG:False
    =>FRAME_SIZE:(1944, 2592)
    =>VIEW_LABEL:ManualCV
    =>OCTAGON_FIBERS:('10/130(oc)', '20/130(oc)')
    =>CAPILLARY:('capillary',)
    =>DYNAMIC_CAMERA:True
    =>SAVED_VIEW_ITEMS:{}
    =>BASE_DIR:D:\MyProjects\WorkProject\opencv4fiber\geomety
2017-10-23 12:50:00,325-__main__-ERROR- main info: ManualCV cvonline 
D:/MyProjects/WorkProject/opencv4fiber/geomety/main.py
2017-10-23 12:50:01,430-setting.parameter-ERROR-key update :
    =>cladRange:[500, 900]
    =>coreRange:[20, 140]
    =>fiberType:G652
    =>corepoint:[1296, 972]
    =>thresholdSize:{u'core': 60, u'clad': 175}
    =>ampPixSize:0.0792393
    =>imgsize:[1944, 2592]
    =>medianBlur:{u'corefilter': 15, u'cladfilter': False}
2017-10-23 12:50:01,430-pattern.classify-ERROR-meta_logger-amp:0.0792-thr:60,175-filter:15,False
2017-10-23 12:50:01,430-pattern.classify-ERROR-get fiber type: G652 - <pattern.classify.DoubleCircleClassify object at 0x08C3B630>
2017-10-23 12:50:05,773-GUI.controller-ERROR-close controller
